app/helpers/es.py

```diff
@@ -68,6 +68,14 @@ def init_connection(self):
 
         return result
 
+    def flush(self):
```

Comment:
You can just use the "flush_bulk_actions" externally. Generally, if a method doesn't start with "_" it's not an internal method and can thus be used outside of the class. Adding another flush method adds unneeded complexity